### PR TITLE
[3.0] Regenerate device listing

### PR DESCRIPTION
**Pull request by krwq**
Fixes: https://github.com/dotnet/iot/projects/4#card-26277821
This depends on https://github.com/dotnet/iot/pull/710

### CODE REVIEW

**Comment by joperezr** [comment]
Does this include the new mcp changes on your other PR?

**Comment by joperezr** [comment]
Oh I just saw your comment about this depending on that other one

**Comment by krwq** [comment]
I wasn't planning to add both commits here, will fix it up soonish

File: src/devices/Device-Index.md [hunk 1]
```diff
@@ -2,51 +2,57 @@
...
-* [SHT3x - Temperature & Humidity Sensor](Sht3x/README.md)
-* [Si7021 - Temperature & Humidity Sensor](Si7021/README.md)
-* [SocketCan - CAN BUS library (Linux only)](SocketCan/README.md)
-* [Software implementation of SPI](SoftwareSpi/README.md)
-* [Software PWM](SoftPwm/README.md)
-* [Solomon Systech Ssd1306 OLED display](Ssd13xx/README.md)
-* [TCS3472x Sensors](Tcs3472x/README.md)
-* [VL53L0X - distance sensor](Vl53L0X/README.md)
-* [Ws28xx LED drivers](Ws28xx/README.md)
+* [28BYJ-48 Stepper Motor 5V 4-Phase 5-Wire & ULN2003 Driver Board](Uln2003\README.md)
```
Comment (krwq) [inline src/devices/Device-Index.md]: FYI: Switching to rich diff makes it easier to read

File: src/devices/Device-Index.md [hunk 2]
```diff
@@ -2,51 +2,57 @@
...
+* [HMC5883L - 3 Axis Digital Compass](Hmc5883l\README.md)
+* [HTS221 - Capacitive digital sensor for relative humidity and temperature](Hts221\README.md)
+* [LM75 - Digital Temperature Sensor](Lm75\README.md)
+* [LPS25H - Piezoresistive pressure and thermometer sensor](Lps25h\README.md)
+* [LSM9DS1 - 3D accelerometer, gyroscope and magnetometer](Lsm9Ds1\README.md)
+* [MAX44009 - Ambient Light Sensor](Max44009\README.md)
+* [Max7219 (LED Matrix driver)](Max7219\README.md)
+* [Mcp23xxx - I/O Expander device family](Mcp23xxx\README.md)
+* [Mcp25xxx device family - CAN bus](Mcp25xxx\README.md)
+* [Mcp3428 - Analog to Digital Converter (I2C)](Mcp3428\README.md)
```
Comment (krwq) [inline src/devices/Device-Index.md]: @Frankenslag seems Mcp3428 can probably also reuse the stuff you wrote

Comment (Frankenslag) [reply src/devices/Device-Index.md]: @krwq 
I think that the differences between i2c and spi will make that difficult.

The 3428 binding is a number of chips and so could have been called Mcp342x

**Pull request merged**